platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r85/ProblemsServiceModelBuilderCrossVersionTest.groovy

```diff
@@ -85,78 +87,21 @@ class ProblemsServiceModelBuilderCrossVersionTest extends ToolingApiSpecificatio
         """
     }
 
-    @TargetGradleVersion(">=8.6")
-    @ToolingApiVersion(">=8.6")
-    def "Can use problems service in model builder"() {
-        given:
-        Assume.assumeTrue(javaHome != null)
-        withSampleProject(false, true)
-        ProblemProgressListener listener = new ProblemProgressListener()
-
-        when:
-        withConnection {
-            it.model(CustomModel)
-                .setJavaHome(javaHome.javaHome)
-                .addProgressListener(listener)
-                .get()
-        }
-        def problems = listener.problems.collect{ it as ProblemDescriptor}
-
-        then:
-        problems.size() == 1
-        problems[0].label.label == 'label'
-        problems[0].category.category == 'testcategory'
-//        problems[0].exception.exception.message == 'test'
-
-        where:
-        javaHome << [
-            AvailableJavaHomes.jdk8,
-            AvailableJavaHomes.jdk17,
-            AvailableJavaHomes.jdk21
-        ]
-    }
-
-    @ToolingApiVersion("<8.6")
     @TargetGradleVersion("<8.6")
-    def "Can use problems service in model builder before 8"() {
+    def "Can use problems service in model builder before 8.6"() {
         given:
         withSampleProject()
         ProblemProgressListener listener = new ProblemProgressListener()
 
         when:
         withConnection {
             it.model(CustomModel)
-                .setJavaHome(javaHome)
-                .addProgressListener(listener)
-                .get()
-        }
-        def problems = listener.problems.collect { new JsonSlurper().parse(listener.problems[0].json.bytes) }
-
-        then:
-        problems.size() == 1
-        problems[0].label == 'label'
-        problems[0].problemCategory == 'testcategory'
-    }
-
-    @ToolingApiVersion(">=8.6")
-    @TargetGradleVersion(">=8.6")
-    def "Can add additional metadata"() {
-        given:
-        withSampleProject(true, true)
-        ProblemProgressListener listener = new ProblemProgressListener()
-
-        when:
-        withConnection { connection ->
-            connection.model(CustomModel)
+                .setJavaHome(jdk17.javaHome)
                 .addProgressListener(listener)
                 .get()
         }
-
         then:
         listener.problems.size() == 1
-        listener.problems[0].additionalData.asMap == [
-            'keyToString': 'value'
-        ]
     }
 
     @TargetGradleVersion(">=8.6")
```

Comment:
❌ This doesn't seem right. I'm getting lost in the test reshuffling, but I assume we should have something like 

```
    @ToolingApiVersion(">=8.6")
    @TargetGradleVersion(">=8.5 <8.7")
    def "getFailure always null in older versions"() { ... }
```

in the `r86` package.